scm/src/gmtb_scm_input.f90

```diff
@@ -180,22 +188,68 @@ subroutine get_case_init(scm_state, scm_input)
   type(scm_input_type), target, intent(inout) :: scm_input
 
   integer               :: input_nlev !< number of levels in the input file
+  integer               :: input_nsoil !< number of soil levels in the input file
   integer               :: input_ntimes !< number of times represented in the input file
 
   ! dimension variables
   real(kind=dp), allocatable  :: input_pres(:) !< input file pressure levels (Pa)
   real(kind=dp), allocatable  :: input_time(:) !< input file times (seconds since the beginning of the case)
 
   !initial profile variables
-  real(kind=dp), allocatable  :: input_height(:) !< height of the pressure levels
   real(kind=dp), allocatable  :: input_thetail(:) !< ice-liquid water potential temperature profile (K)
```

Comment:
Note to self: investigate whether ice-liquid water potential temperature should be the expected input. Temperature is more familiar to potential users and is what is available for FV3 ICs, apparently. It can also just be constructed from T, p, ql, qi. However, many cases specify their thermodynamic state in theta, theta_l, or theta_il. Maybe support both as is done here?

Reply:
Actually, the FV3 initial conditions only have geopotential, and I calculate temperature from that.
I initially tried to calculate ice-liquid water potential temperature, but I couldn't reproduce the full model's physics tendencies, so I switched to temperature.